### PR TITLE
Document migration from scenarios to DAML script

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: docs/source/daml-script/index.rst [hunk 1]
```diff
@@ -199,6 +199,30 @@ We can then initialize our ledger passing in the json file via ``--input-file``.
 
 If you open Navigator, you can now see the contracts that have been created.
 
+Migrating from Scenarios to DAML Script for Ledger Initialization
+=================================================================
+
+Existing scenarios that you used for ledger initialization can be
+translated to DAML script but there are a few things to keep in mind:
```
Comment (aherrmann-da) [inline docs/source/daml-script/index.rst]: Should there be a separate section introducing `init-script` as a concept before going into migration, or is this planned for later when DAML script is stabilized?

Comment (cocreature) [reply docs/source/daml-script/index.rst]: Good point! I’ve added a section on that. PTAL

Comment (aherrmann-da) [reply docs/source/daml-script/index.rst]: Thank you, that looks great!

File: docs/source/daml-script/index.rst [hunk 2]
```diff
@@ -196,6 +196,49 @@ We can then initialize our ledger passing in the json file via ``--input-file``.
...
 If you open Navigator, you can now see the contracts that have been created.
 
+Using DAML Script for Ledger Initialization
+===========================================
+
+You can use DAML script to initialize a ledger on startup. To do so,
+specify an ``init-script: ScriptExample:initializeFixed`` field in
+your ``daml.yaml``. This will automatically be picked up by ``daml
+start`` and used to initialize sandbox. Since it is often useful to
+create a party with a specific party id during development, you can
```
Comment (hurryabit) [inline docs/source/daml-script/index.rst]:
```suggestion
create a party with a specific party identifier during development, you can
```
Same below.

File: docs/source/daml-script/index.rst [hunk 3]
```diff
@@ -196,6 +196,49 @@ We can then initialize our ledger passing in the json file via ``--input-file``.
...
+   ``createAndExercise`` need to be suffixed with ``Cmd``, e.g.,
+   ``createCmd``.
+#. Instead of specifying a ``scenario`` field in your ``daml.yaml``,
+   you need to specify an ``init-script`` field. The initialization
+   script is specified via ``Module:identifier`` for both fields.
+#. DAML script only supports the commands available on the ledger API
+   so you cannot call functions like ``fetch`` directly. This is
+   intentional. Your initialization scripts should not be able to
+   create transactions that a ledger client would not be able to
+   create. If you need, you can create a choice and call that via
```
Comment (hurryabit) [inline docs/source/daml-script/index.rst]: What do you mean by "create a choice"? I suppose you mean something like creating a contract with a single choice?

Comment (cocreature) [reply docs/source/daml-script/index.rst]: Yep, tried to make it a bit clearer in https://github.com/digital-asset/daml/pull/4870

**Pull request merged**